shardmanager.go

```diff
@@ -31,6 +31,12 @@ type ShardManager struct {
 const listeningShardTopic = "listeningShard"
 const collationTopicFmt = "shardCollations_%d"
 
+const (
```

Comment:
This should be a temporary solution. We should not have knowledge of data type here

Reply:
Added `FIXME`